WooCommerce/Classes/ViewRelated/Products/Variations/Bulk Update/BulkUpdateViewModel.swift

```diff
@@ -21,6 +24,12 @@ final class BulkUpdateViewModel {
     /// The state of synching all variations
     @Published private(set) var syncState: SyncState
 
+    var sections: [Section] {
+        let priceSection = Section(title: Localization.priceTitle, rows: [])
+
+        return [priceSection]
+    }
+
```

Comment:
Should we add unit tests for these changes?

Reply:
Yes, I updated the PR and added `test_number_of_sections()` & `func test_first_section_title()` unit test methods for this.